src/components/Upload.react.js

```diff
@@ -1,4 +1,5 @@
-import React, {Component, PropTypes} from 'react';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
```

Comment:
a few nit-picky things. I have the personal convention of using `{Component}` rather than `{ Component }`